### PR TITLE
M2: Fraud updates

### PR DESCRIPTION
**Pull request by Locrian24**
## Description of the change

> Updates to status cell components and `sqm-referred-registration` to stop help fraudulent referrals.

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation or Development tools (readme, specs, tests, code formatting)

## Links

- Jira issue number: (PUT IT HERE)
- Process.st launch checklist: (PUT IT HERE)

### Development

- [ ] [Prettier](https://www.npmjs.com/package/prettier) was run (if applicable)
- [ ] The behaviour changes in the pull request are covered by specs
- [ ] All tests related to the changed code pass in development

### Paperwork

- [ ] This pull request has a descriptive title and information useful to a reviewer
- [ ] This pull request has a Jira number
- [ ] This pull request has a Process.st launch checklist

### Code review

- [ ] Changes have been reviewed by at least one other engineer
- [ ] Security impacts of this change have been considered

### CODE REVIEW

File: packages/mint-components/src/components/sqm-referral-table/cells/sqm-referral-table-rewards-cell.tsx [hunk 1]
```diff
@@ -143,6 +156,9 @@ export class ReferralTableRewardsCell {
           status: state,
         }
       );
+
+      console.log({ statusText, longText: this.statusLongText, state });
```
Comment (noahwc) [inline packages/mint-components/src/components/sqm-referral-table/cells/sqm-referral-table-rewards-cell.tsx:160]: Remove `console.log`

File: packages/mint-components/package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@saasquatch/mint-components",
   "title": "Mint Components",
-  "version": "1.6.18",
+  "version": "1.8.0-11",
```
Comment (noahwc) [inline packages/mint-components/package.json:4]: Bump to a non-prerelease version

Comment (noahwc) [inline packages/mint-components/package.json:4]: Bump to a non prerelease version

File: packages/mint-components/src/components.d.ts [hunk 3]
```diff
@@ -1825,6 +1831,8 @@ export namespace Components {
     }
     interface SqmReferralTableStatusCell {
         "converted": boolean;
+        "fraudStatus"?: "APPROVED" | "PENDING" | "DENIED";
```
Comment (noahwc) [inline packages/mint-components/src/components.d.ts:1834]: Consider creating a fraud status type since this literal union is used in a few places.

**Review by noahwc — APPROVED** [review]

**Review by noahwc — APPROVED** [review]

File: packages/bedrock-components/CHANGELOG.md [hunk 4]
```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.4.0] - 2023-01-08
+- `@saasquatch/component-boilerplate` package bump
```
Comment (00salmon) [inline packages/bedrock-components/CHANGELOG.md:11]:
```suggestion
## [1.4.0] - 2023-01-08

- `@saasquatch/component-boilerplate` package bump

```
just for consistent spacing

File: packages/component-boilerplate/CHANGELOG.md [hunk 5]
```diff
@@ -5,6 +5,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.6.0] - 2023-01-08
+
+### Added
+- Fraud check in `useAuthenticateManagedIdentityWithInstantAccess` hook
+
```
Comment (00salmon) [inline packages/component-boilerplate/CHANGELOG.md:12]:
```suggestion
### Added

- Fraud check in `useAuthenticateManagedIdentityWithInstantAccess` hook

```
also spacing

File: packages/mint-components/CHANGELOG.md [hunk 6]
```diff
@@ -7,6 +7,36 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+    - Added `pendingReviewText` string prop
+    - Added `deniedText` string prop
+  - \<sqm-rewards-table-status-column>
+    - Added support for `PENDING_REVIEW` and `DENIED` statuses in `statusText` string prop
+    - Added support for `PENDING_REVIEW` and `DENIED` statuses in `statusLongText` string prop
+    - Added `pendingReviewText` string prop
+    - Added `deniedText` string prop
+  - \<sqm-referred-registration>
+    - Added a fraud check post-registration
+    - A user will now not be given access if the referral created upon registration was deemed fraudulent
```
Comment (00salmon) [inline packages/mint-components/CHANGELOG.md:38]:
```suggestion

### Added

- Added support for fraud referral moderation to multiple components

### Changed

- Changed components:
  - \<sqm-referral-table-status-cell>
    - Added optional `fraudStatus` enum prop
    - Added `statusSubText` string prop
  - \<sqm-referral-table-status-column>
    - Added `pendingReviewStatusText` string prop
    - Added `deniedStatusText` string prop
    - Added `pendingReviewStatusSubText` string prop
    - Added `deniedStatusSubText` string prop
  - \<sqm-referral-table-rewards-cell>
    - Added `deniedHelpText` string prop
  - \<sqm-referral-table-rewards-column>
    - Added support for `PENDING_REVIEW` and `DENIED` statuses in `statusText` string prop
    - Added support for `PENDING_REVIEW` and `DENIED` statuses in `statusLongText` string prop
    - Added `deniedHelpText` string prop
  - \<sqm-rewards-table-status-cell>
    - Added optional `fraudStatus` enum prop
    - Added `pendingReviewText` string prop
    - Added `deniedText` string prop
  - \<sqm-rewards-table-status-column>
    - Added support for `PENDING_REVIEW` and `DENIED` statuses in `statusText` string prop
    - Added support for `PENDING_REVIEW` and `DENIED` statuses in `statusLongText` string prop
    - Added `pendingReviewText` string prop
    - Added `deniedText` string prop
  - \<sqm-referred-registration>
    - Added a fraud check post-registration
    - A user will now not be given access if the referral created upon registration was deemed fraudulent
```
more changelog formatting

**Review by 00salmon — APPROVED** [review]
🚀

**Review by 00salmon — APPROVED** [review]
🚀 🕐

**Pull request merged**